crates/air/src/commands/format.rs

```diff
@@ -255,3 +255,50 @@ impl Display for FormatCommandError {
         }
     }
 }
+
+#[cfg(test)]
+mod test {
```

Comment:
These moved from integration tests back to being unit tests. And they can also now live in the same file (previously the global logger was preventing that)